tnb/banks.py

```diff
@@ -53,6 +53,25 @@ def patch_account(self, account_number, node_id, trust, signature):
 
         return self.patch(resource, body=body)
 
+    def patch_validator(self, node_id, trust, signature):
+        """
+        Send a PATCH request of an account to a Bank
+
+        :param node_identifier: Node identifier of the bank
```

Comment:
let's keep the comments consistent with the naming convention. Also, I think you are missing some params

Reply:
Off, my bad. Fixed it with d4f6220